packages/manager/modules/pci/src/projects/project/storages/databases/database/metrics/metrics.controller.js

```diff
@@ -106,8 +106,19 @@ export default class MetricsCtrl {
         availableMetric,
         this.selectedTimeRange.value,
       ).then((data) => {
-        const metrics = sortBy(data.metrics, 'hostname');
-        this.metricsData[data.name].data = data;
+        const metricData = data;
+        if (['net_send', 'net_receive'].includes(availableMetric)) {
+          metricData.metrics[0].dataPoints = data.metrics[0].dataPoints.map(
+            (point) => {
+              return {
+                timestamp: point.timestamp,
+                value: point.value / 1000000,
```

Comment:
**suggestion:** Better to add it as a constant with a meaningful name which will result in better understanding.

Reply:
I have updated the code to use constants, it looks cleaner :smile: https://github.com/ovh/manager/pull/7124/commits/b0992dc00c31840deffebd72d3358801e92a6bbb